src/components/db.ts

```diff
@@ -114,7 +114,8 @@ const initUserProfileTable = async (db: Database): Promise<void> => {
             last_updated TIMESTAMP NOT NULL DEFAULT CURRENT_DATE,
             faculty VARCHAR(32),
             program VARCHAR(32),
-            specialization VARCHAR(32)
+            specialization VARCHAR(32),
+            profile_emoji VARCHAR(32)
```

Comment:
At the moment, if `user_profile` already exists (without the new profile_emoji column), this will not work. The table would not be created (since it already exists), which means the new column would not be added either.

This is more so an issue with Codey's underlying design - we should make the addition of columns more dynamic when we create our tables. In the meantime, there seems to be some resources here (https://stackoverflow.com/questions/3604310/alter-table-add-column-if-not-exists-in-sqlite#:~:text=Unfortunately%2C%20SQLite%20doesn't%20support,example%20(but%20in%20C%2B%2B) that you could check out for how to resolve this.

Reply:
Added the logic to check for existence of column